ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-security.j2

```diff
@@ -0,0 +1,58 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - ip security #}
+{% if ip_security is arista.avd.defined %}
+!
+ip security
+{%     for ike_policy in ip_security.ike_policies | arista.avd.default([]) %}
+   ike policy {{ ike_policy.name }}
+{%         if ike_policy.local_id is arista.avd.defined %}
+      local_id {{ ike_policy.local_id }}
+   !
```

Comment:
Looks wrong with a ! after the policy. Is that correct? I mean if ike policy is alone, it would be wrong? Normally we put such things above the section. So you could remove it here and move it the the beginning of your sa_policy loop. Same for profile.

Reply:
yes - will do this! thanks - just the first policy looks like this:
```
ip security
   ike policy IKE-P-1
   !
   ike policy IKE-P-2
   !
   sa policy SA-P-1
   ```
   
   so we would end up with an extra `!`

Reply:
Addressed in [8053791](https://github.com/aristanetworks/ansible-avd/pull/3194/commits/80537919832da472d38eaff495688e138a5e88e5)